tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/unit_normalization_test.py

```diff
@@ -1,36 +1,138 @@
+      measurement_source_value INT64,
+      measurement_source_concept_id INT64,
+      unit_source_value INT64,
+      value_source_value INT64
+      >>
+    -- 3020509 Albumin/Globulin [Mass Ratio] in Serum or Plasma https://athena.ohdsi.org/search-terms/terms/3020509 --
+    -- 3020891 Body temperature https://athena.ohdsi.org/search-terms/terms/3020891 --
+    -- 3016293 Bicarbonate [Moles/volume] in Serum or Plasma https://athena.ohdsi.org/search-terms/terms/3016293 --
+    -- 3027970 Globulin [Mass/volume] in Serum by calculation https://athena.ohdsi.org/search-terms/terms/3027970 --
+    -- 3000963 Hemoglobin [Mass/volume] in Blood https://athena.ohdsi.org/search-terms/terms/3000963 --
+    -- 000905 Leukocytes [#/volume] in Blood by Automated count https://athena.ohdsi.org/search-terms/terms/3000905 --
+    -- 3020630 Protein [Mass/volume] in Serum or Plasma https://athena.ohdsi.org/search-terms/terms/3020630 --
+    -- 3020416 Erythrocytes [#/volume] in Blood by Automated count https://athena.ohdsi.org/search-terms/terms/3020416 --
+      [
+      (1,1,3020509,0.4,8523,1.0,2.4,0,0,0,0,0,0,0,0,0,0,0),
+      (2,1,3020891,97.7,9289,0.0,150.0,0,0,0,0,0,0,0,0,0,0,0),
+      (3,1,3016293,25.0,8753,21.0,31.0,0,0,0,0,0,0,0,0,0,0,0),
+      (4,1,3027970,2.3,8713,1.5,4.5,0,0,0,0,0,0,0,0,0,0,0),
+      (5,1,3027970,2.6,4121395,1.9,3.7,0,0,0,0,0,0,0,0,0,0,0),
+      (6,1,3000963,15.8,4121395,13.2,17.1,0,0,0,0,0,0,0,0,0,0,0),
+      (7,1,3000905,10.1,8647,4.5,11.0,0,0,0,0,0,0,0,0,0,0,0),
+      (8,1,3020630,6.2,8840,6.4,8.2,0,0,0,0,0,0,0,0,0,0,0),
+      (9,1,3020416,5.06,8816,4.2,5.8,0,0,0,0,0,0,0,0,0,0,0),
+      (10,1,3000963,3.4,8554,0.5,5.0,0,0,0,0,0,0,0,0,0,0,0)] col
+)
+SELECT
+    measurement_id,
+    person_id,
+    measurement_concept_id,
+    value_as_number,
+    unit_concept_id,
+    range_low,
+    range_high,
+    measurement_date,
+    measurement_datetime,
+    measurement_type_concept_id,
+    operator_concept_id,
+    value_as_concept_id,
+    provider_id,
+    visit_occurrence_id,
+    measurement_source_value,
+    measurement_source_concept_id,
+    unit_source_value,
+    value_source_value
+  FROM
+    w,
+    UNNEST(w.col))
+""")
+
+
+class UnitNormalizationTest(BaseTest.CleaningRulesTestBase):
 
     @classmethod
     def setUpClass(cls):
         print('**************************************************************')
         print(cls.__name__)
         print('**************************************************************')
 
-    def setUp(self):
+        super().initialize_class_vars()
+
         # Set the test project identifier
-        project_id = os.environ.get(PROJECT_ID)
-        self.project_id = project_id
+        cls.project_id = os.environ.get(PROJECT_ID)
 
         # Set the expected test datasets
-        self.dataset_id = os.environ.get('COMBINED_DATASET_ID')
-        self.sandbox_id = self.dataset_id + '_sandbox'
+        cls.dataset_id = os.environ.get('COMBINED_DEID_DATASET_ID')
```

Comment:
Locally I had to set the environment variable `COMBINED_DEID_DATASET_ID` but I wonder if it's failing because that's not set in CircleCI? Maybe change this to `COMBINED_DATASET_ID` and see if it will pass?